src/Microsoft.Diagnostics.DebugServices.Implementation/ServiceManager.cs

```diff
@@ -345,13 +356,14 @@ private sealed class ExtensionLoadContext : AssemblyLoadContext
                 "SOS.InstallHelper"
             };
 
-            private Dictionary<string, string> _extensionPaths;
+            private static readonly string _defaultAssembliesPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
Is ExecutingAssembly guaranteed under all the cases to be where to look? The exe host could be windbg or VS. Is the right check perhaps the folder of *this* assembly?

Reply:
Eh, I doubt it's faster - I expect it to also add memory per ALC. But given the current usage either is fine.